pkgs/tools/system/netdata/default.nix

```diff
@@ -1,18 +1,20 @@
-{ stdenv, callPackage, fetchFromGitHub, autoreconfHook, pkgconfig
+{ stdenv, callPackage, fetchFromGitHub, makeWrapper, autoreconfHook, pkgconfig
 , CoreFoundation, IOKit, libossp_uuid
 , curl, libcap,  libuuid, lm_sensors, zlib, fetchpatch
 , withCups ? false, cups
 , withDBengine ? true, libuv, lz4, judy
 , withIpmi ? (!stdenv.isDarwin), freeipmi
 , withNetfilter ? (!stdenv.isDarwin), libmnl, libnetfilter_acct
 , withSsl ? true, openssl
+, withCloud ? true, libwebsockets_3_2, json_c
```

Comment:
The cloud is unfree and should probably not be enabled when the user does not allow unfree packages.

Reply:
If I understand correctly the agent-side of the cloud code is released under GPLv3, whereas the server part of Netdata Cloud isn't released at all.

Reply:
I can confirm that the client-side Netdata code is 100% Open Source while the server-side is not. Nixpkgs already contains other packages with this same arrangement (hercules-ci-agent, cachix and the GitHub CLI, just from the top of my head) and they are generally considered "free".